linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_tally2_unlock_time/oval/shared.xml

```diff
@@ -0,0 +1,46 @@
+<def-group>
+  <definition class="compliance" id="{{{ rule_id }}}" version="1">
+    {{{ oval_metadata("The unlock time after number of failed logins should be set correctly.") }}}
+    <criteria operator="AND" comment="Checks common to both scenarios">
+      <criterion test_ref="test_accounts_passwords_pam_tally2_unlock_time"
+      comment="Verify unlock time configuation of pam_tally2 in common-auth" />
+      <criterion test_ref="test_accounts_passwords_pam_tally2_unlock_time_account"
+      comment="Verify unlock time configuation of pam_tally2 in common-account" />
+    </criteria>
+
+  </definition>
+
```

Comment:
Just for style purposes, could remove one extra empty line here, please?

Reply:
> Just for style purposes, could remove one extra empty line here, please?

:bow: Done in e3f3597

Reply:
:bow: Done in e3f3597